app/models/mixins/archived_mixin.rb

```diff
@@ -4,6 +4,11 @@ module ArchivedMixin
   included do
     scope :archived, -> { where.not(:deleted_on => nil) }
     scope :active, -> { where(:deleted_on => nil) }
+
+    # records are active from the date where they were created until the date they are deleted
+    scope :active_for, (lambda do |timestamp|
+      unscope(:where => :deleted_on).where(arel_table[:deleted_on].eq(nil).or(arel_table[:deleted_on].gteq(timestamp)))
+    end)
```

Comment:
This lambda in parens thing looks weird to be, when we typically use `->` notation.  Can we do the following?

```suggestion
    scope :active_for, ->(timestamp) {
      unscope(:where => :deleted_on).where(arel_table[:deleted_on].eq(nil).or(arel_table[:deleted_on].gteq(timestamp)))
    }
```

Reply:
I has `->` and rubocop asked me to use lambda.

maybe drop the `()`?

Reply:
I tried other formats, and that is what rubocop wants. The parens on the outside, the `do`/`end`, and the `lambda`

I'm updating to the format you suggested since that is what I originally had, and I agree, looks better.

maybe we'll update rubocop some day

> `Style/Lambda: Use the lambda method for multiline lambdas`

Reply:
Then we should change rubocop haha - it's fine, then - we can worry about it later.